service/collector/key/key.go

```diff
@@ -0,0 +1,13 @@
+package key
+
+func TillerDeploymentName() string {
+	return "tiller-deploy"
+}
+
+func TillerMaxHistoryEnvVarName() string {
+	return "TILLER_HISTORY_MAX"
+}
+
+func TillerMaxHistoryEnvVarValue() string {
+	return "10"
```

Comment:
```suggestion
	const (
	  TillerDeploymentName = "tiller-deploy"
	)
```
Does it make sense to use `const` rather than function?

Reply:
The key package uses the same approach we use in e2e. I don't think using constants adds much value.

https://github.com/giantswarm/aws-operator/blob/master/integration/key/release_name.go